libs/application/templates/parental-leave/src/fields/Rights/RequestRightsRadio.tsx

```diff
@@ -5,14 +5,18 @@ import {
   FieldBaseProps,
   FieldComponents,
   FieldTypes,
+  getErrorViaPath,
   RadioField,
 } from '@island.is/application/core'
 import { RadioFormField } from '@island.is/application/ui-fields'
-
+import {
+  NO,
+  YES,
+  YesOrNo,
+  getApplicationAnswers,
+} from '@island.is/application/templates/parental-leave'
```

Comment:
It's not recommended to import from self like this, as it can cause circular dependency errors.